02-KernelModules_CharacterDevice/Makefile

```
@@ -0,0 +1,18 @@
+TARGET = character_device
+
+
+ifneq ($(KERNELRELEASE),)
+
+obj-m := $(TARGET).o
+$(TARGET)-objs := main.o
+
+else
+
+KERNELDIR := $(BUILD_KERNEL)
+
+all:
+	$(MAKE) -C $(KERNELDIR) M=$(CURDIR) modules
+clean:
+	$(MAKE) -C $(KERNELDIR) M=$(CURDIR) clean
+
+endif
```

[02-KernelModules_CharacterDevice/Makefile](https://github.com/Kernel-GL-HRK/gl-kernel-basecamp-2019/pull/106/files#diff-966f94605c34f5523e4fc8ce26c2ca05)

```suggestion
endif

```

Please, add empty line at the end of file